kubernetes/linux/main.sh

```diff
@@ -263,15 +263,33 @@ if [[ ((! -e "/etc/config/kube.conf") && ("${CONTAINER_TYPE}" == "PrometheusSide
       fi
 fi
```

Comment:
please merge latest changes from ci_prod to avoid any conflicts when you merge to ci_prod.

Reply:
Will merge after both the PRs are completed